object/application.go

```diff
@@ -372,7 +373,7 @@ func GetMaskedApplications(applications []*Application, userId string) []*Applic
 	return applications
 }
 
-func UpdateApplication(id string, application *Application) (bool, error) {
+func UpdateApplication(ctx *context.Context, id string, application *Application) (bool, error) {
```

Comment:
why context arg with pointer?

Reply:
another point - this is Beego context, Beego library struct. it's bad dependency for object layer, I think we should avoid this, could you try to find solution with context.Context(go built-in  interface) with record link inside as arg, or with record Object as arg?

Reply:
fixed